docs/fsharp-cheatsheet.md

```diff
@@ -404,6 +403,31 @@ Another way of implementing interfaces is to use *object expressions*.
 
 <a name="ActivePatterns"></a>Active Patterns
 ---------------
+*Single-case active patterns*:
+
+    // Basic
+    let (|EmailDomain|) (email: string) =
+        let parts = email.Split '@'
+        parts[1]
```

Comment:
This can be addressed later but we should make this real by handling cases where there's no `@`
There's a .NET class encapsulating email addresses too,  but using it in a cheatsheet might be a bit much